src/routes/routes.js

```diff
@@ -66,5 +68,8 @@ router.patch('/api/v1/profiles/:username/notifications/:subscribe', [verifyToken
 router.get('/api/v1/profiles/notifications/all', verifyToken, Notification.getNotification);
 router.patch('/api/v1/profiles/notifications/:id/read', verifyToken, Notification.readOneNotification);
 router.patch('/api/v1/profiles/notifications/read/all', verifyToken, Notification.readAllNotification);
+router.get('/api/v1/bookmarks/:slug', [verifyToken, findUser], Bookmark.createBookmark);
```

Comment:
since we're creating a resource `POST` is a more suitable verb even though we are not sending any data

Reply:
Thought of it that way, let me correct it right away.